landoapi/workers/landing_worker.py

```diff
@@ -422,6 +422,7 @@ def run_job(
                 return False  # Try again, this is a temporary failure.
             except Exception as e:
                 message = f"Unexpected error while pushing to {repo.push_path}.\n{e}"
+                logger.exception(message)
```

Comment:
The error related to this change happened in the section above this one (where `False` is returned, not `True`.)